templates/appliance/shared/_intel-nuc.html

```diff
@@ -6,7 +6,7 @@
   <div class="row u-vertically-center">
     <div class="col-8">
       {% if appliance.iso_url.pc %}
-      <h1>Install the Ubuntu {{ appliance.name }} appliance for Intel NUC</h1>
```

Comment:
You didn't kill the auto download here

Reply:
ha, no, I was going to add that as an instruction instead... sorry